src/pages/ProjectStart/Components/ItemContents/AddReward.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import styled from 'styled-components';
+import { lighten } from 'polished';
+import TwoButtons from './TwoButtons';
+
+export default function AddReward() {
+  return (
+    <>
+      <List>
+        <h6>선물에 대해 알려주세요</h6>
+        <label htmlFor="rewardName">선물 이름</label>
+        <input
+          id="rewardName"
+          placeholder="선물의 이름을 적어주세요"
+          type="text"
+        />
```

Comment:
```suggestion
        <label>선물 이름
          <input type="text" placeholder="선물의 이름을 적어주세요" />
        </label>
```
- label 태그로 input 태그를 감싼다면 htmlFor와 id 셀렉터를 사용하지 않아도 됩니다

Reply:
수완수완~!